src/schemas/user.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+/**
+ * name: Name of the user
+ * email: Email address of the user
+ * password: Password of the user
+ * admin: Admin status (default false)
+ */
+
+const userSchema = new Schema({
+    name: {
+        type: String,
+        unique: true,
+        required: [true, "Name is required for registeration"]
+    },
+    email: {
+        type: String,
+        required: [true, "Email is required for registeration"],
+        unique: true,
+        match: [/^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/, "This is not a valid email address"]
+    },
+    password: {
+        type: String,
+        required: [true, "Password is required for registeration"]
```

Comment:
## Possible Issue
As this is a website that is going to be deployed, it seems like a stronger restriction would be better in terms of security. Minimum length of the password, and number + alphabet combination requirement could be employed as an improvement.

Reply:
## Accepted
This seems like an issue that I forgot while I was writing the schema for the user accounts. However, it will be modified once I move onto the security part.